CMakeLists.txt

```diff
@@ -215,17 +180,8 @@ option(
 option(WITH_EXAMPLES "Whether to build examples" ON)
 
 option(WITH_METRICS_PREVIEW "Whether to build metrics preview" OFF)
-
-if(WITH_METRICS_PREVIEW)
-  add_definitions(-DENABLE_METRICS_PREVIEW)
```

Comment:
Should this and `ENABLE_LOGS_PREVIEW` below be moved to api/CMakeLists.txt as well?

Reply:
These two definitions are already in `api/CMakeLists.txt`. It's duplicate here.